### DISCUSSION TITLE
Browsing dependencies sources via Telescope

### DISCUSSION THREAD

**Discussion by HerveH44** [LazyVim/LazyVim#3433] [Q&A]
Hello, 

I'm working with Lazyvim on a rust pet project and I'd like to browse the possible traits and look at the structs that implement them. Right now to look at a trait, I need first to import it and then go to the definition. And then I use a plugin to look at the structure of the file to check the different methods.

Is there a way to have telescope look at the dependency sources so I can look for structs or traits without importing them?

**Comment by folke** [comment]
`<leader>cs`

**Reply by folke** [reply]
Sorry, that's probably not what you meant. No idea
